src/test/dynamo/Beyonce.test.ts

```diff
@@ -223,10 +225,15 @@ describe("Beyonce", () => {
     await testInvertedIndexGSI(jayZ)
   })
 
-  it("should write multiple items at once with jayZ", async () => {
+  it("should write multiple items at once in a transact with jayZ", async () => {
     const jayZ = await createJayZ()
     await testBatchWriteWithTransaction(jayZ)
   })
+
+  it("should write multiple items at once with jayZ", async () => {
```

Comment:
We should have a test for returning the unprocessed items that failed as well. Off the top of my head, I don't remember if we have a test mechanism to make queries fail like that, so if we don't and you need help with that, just let me know.